extensions/linz/README.md

```diff
@@ -26,8 +26,8 @@ extension which adds constraints to default STAC schema properties.
 | ----------------------- | ------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
 | title                   | string | **REQUIRED**. Collection title.                                                                                                                                                                                                                                                                                                                  |
 | security_classification | string | **REQUIRED**. New Zealand Government [Security Classification](https://www.digital.govt.nz/standards-and-guidance/governance/managing-online-channels/security-and-privacy-for-websites/foundations/classify-information/). Must be one of `Unclassified`, `IN-CONFIDENCE`, `SENSITIVE`, `RESTRICTED`, `CONFIDENTIAL`, `SECRET` or `TOP-SECRET`. |
-| created                 | string | **REQUIRED**. Created Datetime.                                                                                                                                                                                                                                                                                                                  |
-| updated                 | string | **REQUIRED**. Last updated Datetime.                                                                                                                                                                                                                                                                                                             |
+| created                 | string | **REQUIRED**. Creation Time.                                                                                                                                                                                                                                                                                                                     |
+| updated                 | string | **REQUIRED**. Last Update Time.                                                                                                                                                                                                                                                                                                                  |
```

Comment:
Could we copy the [descriptions from the standard](https://github.com/radiantearth/stac-spec/blob/f81324f712a608d4a319fd0a8738193a43ef1d51/item-spec/common-metadata.md#date-and-time), or reference that section of the standard, for clarity?